ui/selectors/approvals.ts

```diff
@@ -1,19 +1,71 @@
 import { ApprovalControllerState } from '@metamask/approval-controller';
 import { ApprovalType } from '@metamask/controller-utils';
+import { transactionMatchesNetwork } from '../../shared/modules/transaction.utils';
+import { TransactionMeta } from '../../shared/constants/transaction';
+import { getCurrentChainId } from './selectors';
 
 type ApprovalsMetaMaskState = {
   metamask: {
     pendingApprovals: ApprovalControllerState['pendingApprovals'];
+    unapprovedTxs: {
```

Comment:
`unapprovedTxs` doesn't belong `ApprovalsMetaMaskState` but didn't want to create a separate type just for this